aea/cli/test.py

```diff
@@ -374,13 +400,18 @@ def test_package_collection(
     pytest_args: List[str],
     cov: bool,
     covrc_file: Path,
+    authors: Tuple[str, ...],
 ) -> Tuple[List[str], List[Tuple[int, str]]]:
     """Test a collection of packages."""
 
     coverage_data = []
     failures = []
 
     for package_type, package_dir in available_packages:
+        if authors != () and not any(
+            [author in str(package_dir) for author in authors]
+        ):
+            continue
```

Comment:
```suggestion
        if authors != () and not any(
            [author in package_dir.parts for author in authors]
        ):
            continue
```

Reply:
Or we can update 

https://github.com/valory-xyz/open-aea/blob/5405aa3d6b96784ec1e034c88f8a8881c6323228/aea/helpers/dependency_tree.py#L211 

method to return packages only for a specific author and chain them here 

https://github.com/valory-xyz/open-aea/blob/5405aa3d6b96784ec1e034c88f8a8881c6323228/aea/cli/test.py#L232